kafka/README.md

```diff
@@ -57,6 +57,43 @@ The Kafka check does not include any event at this time.
 ### Service Checks
 The Kafka check does not include any service check at this time.
 
+
+
+## Troubleshooting
+### Agent failed to retrieve RMIServer stub
+
+```
+instance #kafka-localhost-<PORT_NUM> [ERROR]: 'Cannot connect to instance localhost:<PORT_NUM>. java.io.IOException: Failed to retrieve RMIServer stub
+```
+
+The Datadog Agent is unable to connect to the Kafka instance to retrieve metrics from the exposed mBeans over the RMI protocol.
+
+Include the following JVM arguments when starting the Kafka instance to solve resolve this issue *(required for Producer, Consumer, and Broker as they are all separate Java instances)*
+```
+-Dcom.sun.management.jmxremote.port=<PORT_NUM> -Dcom.sun.management.jmxremote.rmi.port=<PORT_NUM>
+```
+
+### Producer and Consumer metrics don't appear in my Datadog application
+By default we only collect broker based metrics. 
+
+If you're running Java based Producers and Consumers, uncomment this section of the yaml file and point the Agent to the proper ports to start pulling in metrics, 
+```yaml
+# - host: remotehost
+    # port: 9998 # Producer
+    # tags:
+    # kafka: producer0
+    # env: stage
+    # newTag: test
+    # - host: remotehost
+    # port: 9997 # Consumer
+    # tags:
+    # kafka: consumer0
+    # env: stage
+    # newTag: test
+```
+
+If you are using custom Producer and Consumer clients that are not written in Java and/or not exposing mBeans, having this enabled would still collect zero metrics. To still submit your metrics from your code use dogstatsd.
```

Comment:
Would be nice to add a link to the dogstatsd documentation on the last word of the sentence